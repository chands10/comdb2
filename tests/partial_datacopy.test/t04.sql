create table t {
    schema {
        int a
        int b
        int c
        int d
    }
    keys {
        datacopy(b, b, b, b, c, c, c, c, b, c, b, c, b, d) "a" = a
    }
};$$

select * from comdb2_keys where tablename='t';

select * from comdb2_partial_datacopies where tablename='t';

drop table t;
