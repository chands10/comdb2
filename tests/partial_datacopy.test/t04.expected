(tablename='t', keyname='A', keynumber=0, isunique='Y', isdatacopy='N', ispartialdatacopy='Y', isrecnum='N', condition=NULL, uniqnulls='N')
(tablename='t', keyname='A', columnname='b')
(tablename='t', keyname='A', columnname='c')
(tablename='t', keyname='A', columnname='d')
